shell/edit/logging.banzaicloud.io.output/providers/opensearch.vue

```diff
@@ -45,6 +45,11 @@ export default {
       if (typeof this.value.ssl_version === 'undefined') {
         this.$set(this.value, 'ssl_version', sslVersions[0]);
       }
+
+      // Suppress type name by default
+      if (typeof this.value.suppress_type_name === 'undefined') {
+        this.$set(this.value, 'suppress_type_name', true);
```

Comment:
Enabling this by default for an optional parameter is an interesting mix. This would also enable the property when editing an existing resource. @bashofmann i think you've had experience with this provider, does the change make sense?

Reply:
Hi @richard-cox ! Thanks for your review!

I think it won't affect the behavior of exisiting resource since we have the prerequisite at line 34.

```
if (this.mode === _CREATE) {
```

Reply:
According to the docs (https://docs.fluentbit.io/manual/pipeline/outputs/opensearch#action-metadata-contains-an-unknown-parameter-type) this flag should be `true` for OpenSearch >= 2.0.0.

I think it's fine to have this on for new outputs by default

It would be good to add the info what this flag is about and when it must be on as a help text in the UI as well.

Also this should be added to the ElasticSearch output as well, there it is mandatory to be on for ElasticSearch >= 8.0.0 (https://docs.fluentbit.io/manual/pipeline/outputs/elasticsearch#action-metadata-contains-an-unknown-parameter-type).

Reply:
Thanks for your review! I'll push the changes today.

Reply:
Help text is `Hide type names by default to improve readability.`

Does it make sense? Can you please take a re-review? @bashofmann Thanks!

![image](https://user-images.githubusercontent.com/15041915/236725494-16bde0f8-ff49-4f51-a494-94ab07abdb06.png)

Reply:
There should be a space between `server.` and `You`, so `server. You`. Otherwise looks great.